pulser-core/pulser/backend/noise_model.py

```diff
@@ -67,12 +67,12 @@ class NoiseModel:
             pulses.
         amp_sigma: Dictates the fluctuations in amplitude as a standard
             deviation of a normal distribution centered in 1.
-        dephasing_prob: The probability of a dephasing error occuring.
-        depolarizing_prob: The probability of a depolarizing error occuring.
-        eff_noise_probs: The probability associated to each effective noise
-            operator.
-        eff_noise_opers: The operators for the effective noise model. The
-            first operator must be the identity.
+        dephasing_prob: The rate of a dephasing error occuring (in rad/µs).
+        depolarizing_prob: The rate (in rad/µs) at which a depolarizing
+            error occurs.
+        eff_noise_probs: The rate associated to each effective noise operator
+            (in rad/µs).
```

Comment:
I agree with the switch from probabilities to rates but I don't think we should use the same variables for this. Instead, we should add new `*_rate` variables instead.
Then, the meaning of the `*_prob` variables can be kept but their default should be changed to `None` and the docstring should point out that the variable is deprecated and `*_rate` should be used instead. 
Additionally, if we detect that the value of `*_prob` is not `None` we raise a `DeprecationWarning` and say that the provided value will be converted to a rate.

Reply:
Hey Henrique,
Yes that's exactly what I have in mind, just did not have time to implement it yet hehe